diff --git a/node_modules/dumi/dist/features/sitemap.js b/node_modules/dumi/dist/features/sitemap.js
index df7c619..e076094 100644
--- a/node_modules/dumi/dist/features/sitemap.js
+++ b/node_modules/dumi/dist/features/sitemap.js
@@ -50,7 +50,7 @@ var sitemap_default = (api) => {
     const writeStream = import_fs.default.createWriteStream(import_path.default.join(api.paths.absOutputPath, "sitemap.xml"));
     smis.pipe(writeStream);
     Object.values(api.appData.routes).forEach((route) => {
-      if (!exclude.includes(route.path) && ![":", "*"].some((char) => route.path.includes(char)) && route.path !== "" && !route.isLayout) {
+      if (!exclude.includes(route.path) && ![":", "*"].some((char) => route.path.includes(char)) && !route.isLayout) {
         smis.write({ url: route.path });
       }
     });
